genesis-builder/src/generate.rs

```diff
@@ -18,6 +18,9 @@ struct L2Module {
     structs: BTreeMap<String, Vec<L2Input>>,
     has_fungible_asset: bool,
     has_non_empty_args: bool,
+    has_evm_call: bool,
+    has_evm_view: bool,
```

Comment:
We probably don't need both `has_call` and `has_view`. Generally this is not true, but for L2 contracts one implies the other is missing.

Reply:
You're right, all the L2 contracts have view methods so `has_evm_view` is not needed. `has_evm_call` is needed because `L1BlockNumber.move` only contains view functions.

Fixed in https://github.com/UmiNetwork/op-move/pull/358/commits/694d9e51463aec8a8cccd577141f46c3c2048918